src/pages/studyView/StudyViewPage.tsx

```diff
@@ -108,7 +109,12 @@ export default class StudyViewPage extends React.Component<IStudyViewPageProps,
                 }
 
                 this.store.updateCurrentTab(props.routing.location.query.tab);
-                this.store.updateStoreFromURL(query);
+                const newStudyViewFilter:StudyViewURLQuery = _.pick(props.routing.location.query, ['id', 'studyId', 'cancer_study_id', 'filters', 'filterAttributeId', 'filterValues']);
+
+                if (!_.isEqual(newStudyViewFilter, this.studyViewQueryFilter)) {
+                    this.store.updateStoreFromURL(newStudyViewFilter);
+                    this.studyViewQueryFilter = newStudyViewFilter;
```

Comment:
seems like this property "studyViewQueryFilter" isn't used anywhere?

Reply:
It's used in the if condition to check whether the filter is the same as before. Don't update the store is the filter hasn't change.